protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/StepTools/MixTools/index.tsx

```diff
@@ -74,6 +78,9 @@ export function MixTools(props: StepFormProps): JSX.Element {
   const is96Channel =
     propsForFields.pipette.value != null &&
     pipettes[String(propsForFields.pipette.value)].name === 'p1000_96'
```

Comment:
can we use `.spec.channels` since is8Channel uses that.

```suggestion
    pipettes[String(propsForFields.pipette.value)].spec.channels === 96
```